Fix for vents and disposals

Bunfus opened the pull request:
# About the pull request
This PR connects most vent networks and adds missing disposal pipes on certain maps. It also fixes broken vents and disposal and adds vent and disposal systems on Shivas Snowball (Fixes #11234).

# Explain why it's good for the game
Less map issues and more consistency between maps.

# Testing Photographs and Procedure
<details>
<summary>Screenshots</summary>
Shivas Snowball - added vent and disposal pipes
<img width="960" height="480" alt="Pr3" src="https://github.com/user-attachments/assets/c23d8d0d-ab31-4ec1-b919-35a4a4784e72" />
New Varadero - tweaked layout
<img width="960" height="480" alt="Pr1" src="https://github.com/user-attachments/assets/ff4da8da-7596-416d-a31c-601d389ebfa0" />
Trijent Dam - tweaked layout + connected vents
<img width="960" height="480" alt="Pr2" src="https://github.com/user-attachments/assets/c9c51493-7db2-462f-be58-9a0682c267a3" />
</details>

# Changelog
:cl: Nemesis1337
fix: fixed broken disposals in Trijent medical
imagetweak: moved the sprite of a broken pipe by 1 pixel
mapadd: added vent and disposal pipes on Shivas Snowball
maptweak: simplified layout of vent and disposal pipes on New Varadero
maptweak: connected vent networks on Trijent Dam
/:cl:

Bunfus committed:
Fix for vents and disposals

Bunfus committed:
Misplaced object fixed

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

Bunfus committed:
Minor tweaks and fix to upp hvh insert

Bunfus committed:
Fixup maps in TGM format

46f8e9af81d3: maps/map_files/DesertDam/Desert_Dam.dmm
46f8e9af81d3: maps/map_files/DesertDam/standalone/crashlanding-upp-bar.dmm
46f8e9af81d3: maps/map_files/DesertDam/standalone/landingzone_desertdam_upp_lz1.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus commented:
Fixed everything mentioned above and additionally tweaked that UPP HvH insert on Dam

Bunfus committed:
Merge branch 'master' into disposal-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

Bunfus committed:
Merge branch 'master' into disposal-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

Bunfus committed:
Minor repositions

Bunfus committed:
Fixup maps in TGM format

379dff1cfb55: maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

Bunfus committed:
Merge branch 'cmss13-devs:master' into disposal-tweaks

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Bunfus committed:
Merge branch 'master' into disposal-tweaks

Bunfus committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into disposal-tweaks

Bunfus committed:
Fixup maps in TGM format

b5fdd5218936: maps/map_files/BigRed/BigRed.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Fixed broken stuff after merge

Bunfus committed:
Settings fix

Bunfus committed:
Fixup maps in TGM format

b5fdd5218936: maps/map_files/Ice_Colony_v3/standalone/panic_room_hold.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Bunfus committed:
Merge branch 'master' into disposal-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Bunfus committed:
Merge branch 'master' into disposal-tweaks

Bunfus committed:
Fixup maps in TGM format

f1249a2b91f8: maps/map_files/DesertDam/standalone/crashlanding-upp-bar.dmm
f1249a2b91f8: maps/map_files/DesertDam/standalone/landingzone_desertdam_upp_lz1.dmm
f1249a2b91f8: maps/map_files/DesertDam/standalone/landingzone_desertdam_uscm_lz1.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge branch 'disposal-tweaks' of https://github.com/Bunfus/cmss13 into disposal-tweaks

Bunfus committed:
Updated newly added inserts

Bunfus committed:
Fixup maps in TGM format

cc9c02951c12: maps/map_files/DesertDam/Desert_Dam.dmm
cc9c02951c12: maps/map_files/DesertDam/standalone/landingzone_desertdam_uscm_lz1.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Bunfus committed:
Conflict fix

Bunfus committed:
Overlooked a pipe

Bunfus committed:
Fixup maps in TGM format

8305b47837a2: maps/map_files/DesertDam/Desert_Dam.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Bunfus committed:
Merge remote-tracking branch 'upstream/master' into disposal-tweaks

Detective-Google commented:
This PR has been stale for a few months, you may request it be reopened or create a new one when you bring it up to date and resolve the conflicts.
